// ascii-animals/animals.js
var exports = module.exports = {};

exports.cat = `
                                | \\
                                | |
                                | |
           |\\                   | |
          /, ~\\                / /
         X     \\\`-.....-------./ /
          ~-. ~  ~              |
             \\             /    |
              \\  /_     ___\\   /
              | /\\ ~~~~~   \\ |
              | | \\        || |
              | |\\ \\       || )
             (_/ (_/      ((_/`;

exports.cat_moon = `*     ,MMM8&&&.            *
                  MMMM88&&&&&    .
                 MMMM88&&&&&&&
     *           MMM88&&&&&&&&
                 MMM88&&&&&&&&
                 'MMM88&&&&&&'
                   'MMM8&&&'      *
          |\\___/|     /\\___/\\
          )     (     )    ~( .              '
         =\\     /=   =\\~    /=
           )===(       ) ~ (
          /     \\     /     \\
          |     |     ) ~   (
         /       \\   /     ~ \\
         \\       /   \\~     ~/
  jgs_/\\_/\\__  _/_/\\_/\\__~__/_/\\_/\\_/\\_/\\_/\\_
  |  |  |  |( (  |  |  | ))  |  |  |  |  |  |
  |  |  |  | ) ) |  |  |//|  |  |  |  |  |  |
  |  |  |  |(_(  |  |  (( |  |  |  |  |  |  |
  |  |  |  |  |  |  |  |\\)|  |  |  |  |  |  |
  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |`;

exports.dog_surprise = `
     __/o \\_
     \\____  \\
         /   \\
   __   //\\   \\
__/o \\-//--\\   \\_/
\\____  ___  \\  |
     ||   \\ |\\ |
    _||   _||_||`;

exports.dog = `___
 __/_  \`.  .-"""-.
 \\_,\` | \\-'  /   )\`-')
  "") \`"\`    \\  ((\`"\`
 ___Y  ,    .'7 /|
(_,___/...-\` (_/_/ sk`;

exports.snake = `                              (_)(_)
                             /     \\
                            /       |
                           /   \\  * |
             ________     /    /\\__/
     _      /        \\   /    /
    / \\    /  ____    \\_/    /
   //\\ \\  /  /    \\         /
   V  \\ \\/  /      \\       /
       \\___/        \\_____/`;

exports.snail = `_____
   .'     \`.
  /  .-=-.  \\   \\ __
  | (  C\\ \\  \\_.'')
 _\\  \`--' |,'   _/
/__\`.____.'__.-'`;

exports.lion_head = `_   _
 _/ \\|/ \\_
/\\\\/   \\//\\
\\|/<\\ />\\|/
/\\   _   /\\
\\|/\\ Y /\\|/
 \\/|v-v|\\/
  \\/\\_/\\/`;
